### DISCUSSION TITLE
You were last verified as a student on "blank".

### DISCUSSION THREAD

**Discussion by Tejas-Rao-MN** [community/community#134749] [GitHub Education]
### Hi everyone,

![image](https://github.com/user-attachments/assets/5cac68ea-be16-47a5-a8e2-a577afda55c0)
This is the issue I'm facing and am looking for a solution.
Thanks for the help.

**Comment by hjenryin** [comment]
Hi, I'm facing the exact same issue here. Do you eventually get the benefits? Thanks!

**Reply by hjenryin** [reply]
Quick update, my application got approved on Aug 11 with a green "Approved", and now it's Aug 15. I received a welcome email for GitHub education, and the green badge turned purple, saying "Congrats! Your request for  @hjenryin was approved and your benefits are now available." I still get the "blank" problem here, but I guess that's a bug that doesn't really matter.

**Comment by odelyo** [comment]
Steps to Resolve the Issue:
Check for Outstanding Requests:

The message indicates that you have an outstanding discount request. This might mean that GitHub is still processing your previous request or that there is a pending action you need to take.
Check your GitHub account for any pending notifications or messages regarding this outstanding request.
Contact GitHub Support:

If the issue persists and you are unable to proceed with your application, it’s best to contact GitHub Support directly. Explain the situation and include the details from the screenshot.
You can reach GitHub Support at [GitHub Support](https://support.github.com/contact).
Verify Your Email and School Information:

Ensure that the email associated with your GitHub account is correct and matches your school’s domain. The message confirms that the email is associated with your school, so make sure all details are accurate.
If there are any discrepancies with your school name or email, try updating your information before reapplying.
Wait for Further Communication:

Sometimes, the verification process can take a bit of time. It may be a matter of waiting for GitHub to process your request. Keep an eye on your email and GitHub notifications for any updates.
Reapply If Necessary:

If you receive confirmation that your outstanding request has been processed or rejected, you may need to reapply for the student benefits. Make sure to follow any instructions provided by GitHub.
